schema/vrs.yaml

```diff
@@ -160,12 +162,20 @@ definitions:
   # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
   # SystemicVariation
 
-  AbsoluteAbundance:
+  Abundance:
+    additionalProperties: false
+    description: >-
+      The measure of a quantity of a molecule in a system.
+    oneOf:
+      - $ref: "#/definitions/CopyNumber"
+    discriminator:
+      propertyName: type
+
+  CopyNumber:
```

Comment:
Reminder: We once considered "CopyNumber" as a class name, but we felt that the ambiguity of the term in common use — i.e., as *both* tandem copies and systemic copies — made it ambiguous for us too. I actually prefer "CopyNumber", so the point of this comment is only to remind that we've been here before and made a different decision. (...which might just mean that we were wrong to abandon it last time)

Reply:
Yeah. I think by layering this as a type of abundance lends enough clarity, and "CopyNumber" is definitely more recognizable as a Variation concept than "Abundance".